packages/react-native/src/private/webapis/performance/specs/NativePerformance.js

```diff
@@ -43,15 +45,15 @@ export type PerformanceObserverInit = {
 
 export interface Spec extends TurboModule {
   +now?: () => number;
-  +mark: (name: string, startTime: number) => void;
+  +mark: (name: string, startTime?: number) => NativePerformanceMarkResult;
```

Comment:
I was going to say that we don't need to return the value in this case because we can create the default in JS, but doing it would remove an unnecessary call to `performance.now()` from JS, so I think it's a good thing in that case.

Reply:
Yeah, I wanted to make one native call less by batching here 😄